pkg/cli/bonk.go

```diff
@@ -43,6 +44,25 @@ const (
 ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀`
 )
 
+// getMacAddr attempts to get the MAC address of the user, and return it as a string
+//
+// Side effects: None 
+func getMacAddr() ([]string, error) {
+    ifas, err := net.Interfaces()
+    if err != nil {
+        return nil, err
+    }
+    var as []string
+    for _, ifa := range ifas {
+        a := ifa.HardwareAddr.String()
+        if a != "" {
+            as = append(as, a)
+        }
+    }
```

Comment:
```suggestion
    as := make([]string, 0, len(ifas)) // Preallocate slice for efficiency
    for _, ifa := range ifas {
        if a := ifa.HardwareAddr.String(); a != "" {
            as = append(as, a)
        }
    }
```


- make([]string, 0, len(ifas)) reduces memory reallocations by estimating the required size.
- it avoids unnecessary variable declarations outside the loop.